identity/aziot-identityd/src/identity.rs

```diff
@@ -853,14 +853,6 @@ impl IdentityManager {
 
         // Create new certificate if needed.
         if device_id_cert.is_none() {
-            if let Ok(key_handle) = self.key_client.load_key_pair(identity_pk).await {
```

Comment:
What fix is this? main still has this code.

Reply:
Keyd in this repo doesn't have delete_key_pair, so can't do this.

I think it's fine, it just means that the old key won't be rotated. That's what already happens on release/1.2, so it's equivalent.

Reply:
Ah, makes sense.